File: resources/expected/ros2/building_control_gen_mixed/strict/src/building_control_cpp_pkg/src/base_code/tcp_tempSensor_runner.cpp
#include "building_control_cpp_pkg/user_headers/tcp_tempSensor_src.hpp"

//=================================================
//  D O   N O T   E D I T   T H I S   F I L E
//=================================================

tcp_tempSensor::tcp_tempSensor() : tcp_tempSensor_base()
{
    // Invoke initialize entry point
    initialize();

    PRINT_INFO("tcp_tempSensor infrastructure set up");
}

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto executor = rclcpp::executors::MultiThreadedExecutor();
    auto node = std::make_shared<tcp_tempSensor>();
    executor.add_node(node);
    executor.spin();
    rclcpp::shutdown();
    return 0;
}

File: resources/expected/ros2/python-building_control_gen_mixed/strict/src/building_control_py_pkg/building_control_py_pkg/base_code/tcp_tempSensor_runner.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from rclpy.executors import MultiThreadedExecutor
from building_control_py_pkg.user_code.tcp_tempSensor_src import tcp_tempSensor
#========================================================
# Re-running Codegen will overwrite changes to this file
#========================================================
def main(args=None):
    rclpy.init(args=args)
    node = tcp_tempSensor()
    executor = MultiThreadedExecutor()
    executor.add_node(node)
    executor.spin()
    rclpy.shutdown()

if __name__ == "__main__":
    main()
